bin/build

```diff
@@ -117,13 +117,13 @@ EOF
 
 apt_env_dir=$apt_layer/env
 mkdir -p $apt_env_dir
-echo "$apt_layer/usr/bin:$PATH" > $apt_env_dir/PATH
-echo "$apt_layer/usr/lib/x86_64-linux-gnu:$apt_layer/usr/lib/i386-linux-gnu:$apt_layer/usr/lib" > $apt_env_dir/LD_LIBRARY_PATH
-echo "$apt_layer/usr/lib/x86_64-linux-gnu:$apt_layer/usr/lib/i386-linux-gnu:$apt_layer/usr/lib" > $apt_env_dir/LIBRARY_PATH
-echo "$apt_layer/usr/include:$apt_layer/usr/include/x86_64-linux-gnu" > $apt_env_dir/INCLUDE_PATH
+echo -n "$apt_layer/usr/bin:$PATH" > $apt_env_dir/PATH
+echo -n "$apt_layer/usr/lib/x86_64-linux-gnu:$apt_layer/usr/lib/i386-linux-gnu:$apt_layer/usr/lib" > $apt_env_dir/LD_LIBRARY_PATH
```

Comment:
All of these values should be added to

`${ENV_NAME}.append`. The current code (without the append prefix) will probably lead to some other errors. It should also not use the `$PATH` in L120 because that would add the build time PATH to runtime which may point to entries that do not exist.

Reply:
See https://github.com/buildpacks/spec/blob/main/buildpack.md#environment-variable-modification-rules

Reply:
So how should we add things to PATH for just build time?

Reply:
There's an env.build directory for env variables that are supposed to be build only. You can also export the env var if you only want to make it available to the same buildpack.

Reply:
So I think the idea here is that these layers are also made available at runtime. Given that, what change would we do to the buildpack to support it?

Reply:
I would say use prepend and append env vars instead of overrides.

Reply:
Given the env change, should we drop the profile script?